atom/browser/native_window_mac.mm

```diff
@@ -405,22 +405,28 @@ static bool FromV8(v8::Isolate* isolate, v8::Handle<v8::Value> val,
   // Use an NSEvent monitor to listen for the wheel event.
   BOOL __block began = NO;
   wheel_event_monitor_ = [NSEvent
-    addLocalMonitorForEventsMatchingMask:NSScrollWheelMask
-    handler:^(NSEvent* event) {
-      if ([[event window] windowNumber] != [window_ windowNumber])
-        return event;
-
-      if (!began && (([event phase] == NSEventPhaseMayBegin) ||
-                                 ([event phase] == NSEventPhaseBegan))) {
-        this->NotifyWindowScrollTouchBegin();
-        began = YES;
-      } else if (began && (([event phase] == NSEventPhaseEnded) ||
-                           ([event phase] == NSEventPhaseCancelled))) {
-        this->NotifyWindowScrollTouchEnd();
-        began = NO;
-      }
-      return event;
-  }];
+      addLocalMonitorForEventsMatchingMask:NSScrollWheelMask
+                                   handler:^(NSEvent* event) {
+                                     if ([[event window] windowNumber] !=
+                                         [window_ windowNumber])
+                                       return event;
+
+                                     if (!began && (([event phase] ==
+                                                     NSEventPhaseMayBegin) ||
+                                                    ([event phase] ==
+                                                     NSEventPhaseBegan))) {
+                                       this->NotifyWindowScrollTouchBegin();
+                                       began = YES;
+                                     } else if (began &&
+                                                (([event phase] ==
+                                                  NSEventPhaseEnded) ||
+                                                 ([event phase] ==
+                                                  NSEventPhaseCancelled))) {
+                                       this->NotifyWindowScrollTouchEnd();
+                                       began = NO;
+                                     }
+                                     return event;
+                                   }];
```

Comment:
this is unfortunate. Any way we can convince clang-format to let this callback breathe a little?